### ISSUE TITLE
Broadlink Binding in OH is not coming to lean mode

### ISSUE THREAD

**Issue by scheuerer** [openhab/openhab-addons#15626]
Hello dev's.

I using for testing an OH4.1 Test-Installation with my production setup from 4.0.2
Everything works fine.

Now I have installed the occfical broading binding and instralled an RM4 mini.
The RM4 mini itself works with my prod system under 4.0.2 an the "UNOFFICAL" binding very well.

Now that under 4.1 the item is online I have tryed to bring the RM4 mini into leaning mode:
But this will not work.

Here is the logfile with the error from OH4.1 installation:

`2023-09-19 07:57:10.639 [DEBUG] [nal.handler.RMUniversalRemoteHandler] - Command: ON
2023-09-19 07:57:10.641 [DEBUG] [nternal.handler.BroadlinkBaseHandler] - Authenticating with broadlink device Broadlink RM4...
2023-09-19 07:57:10.669 [WARN ] [com.github.mob41.blapi.BLDevice     ] - RM2 enter learning received error: fffffffb / -5
2023-09-19 07:57:10.671 [DEBUG] [nal.handler.RMUniversalRemoteHandler] - Thing broadlinkthermostat:rmuniversaldevice:08aa07e868 entered learning mode
2023-09-19 07:57:10.612 [INFO ] [openhab.event.ItemCommandEvent      ] - Item 'Broadlink_RM4_Learning_Mode' received command ON
2023-09-19 07:57:10.621 [INFO ] [penhab.event.ItemStatePredictedEvent] - Item 'Broadlink_RM4_Learning_Mode' predicted to become ON
2023-09-19 07:57:10.622 [INFO ] [openhab.event.ItemStateChangedEvent ] - Item 'Broadlink_RM4_Learning_Mode' changed from OFF to ON`

Any idea what is wrong?

**Comment by aalduanc** [comment]
Is there a planned date for the resolution of the bug?

**Comment by lsiepel** [comment]
Not sure if it is related, but the log shows `RM2`  and not `RM4mini`

The binding is also not (yet) part of openHAB. For support, you might need @AntonJansen as he is working towards a merge of the binding into the openHAB repository.

**Comment by AntonJansen** [comment]
Yes, the binding is not part of the distribution yet. Still working on getting the code merged. Please try one of the latest version of the bindings you can find here: https://community.openhab.org/t/broadlink-binding-4-1-0-4-2-0/154734
